DigitalLearningSolutions.Web/Views/Shared/_DlsFooter.cshtml

```diff
@@ -9,6 +9,9 @@
                 <li class="nhsuk-footer__list-item">
                     <a class="nhsuk-footer__list-item-link" target="_blank" asp-controller="LearningSolutions" asp-action="Terms">Terms of use <span class="nhsuk-u-visually-hidden">(opens new browser window)</span></a>
                 </li>
+                <li class="nhsuk-footer__list-item">
+                    <a class="nhsuk-footer__list-item-link" target="_blank" asp-controller="LearningSolutions" asp-action="AcceptableUsePolicy">Acceptable Use Policy <span class="nhsuk-u-visually-hidden">(opens new browser window)</span></a>
```

Comment:
Rename "Acceptable Use Policy" to "Acceptable use policy" in order to be consistent with casing used in other lables in the footer

Reply:
Link text updated. Please review.